module.nix

```diff
@@ -20,23 +20,23 @@ in {
     };
   };
   config = lib.mkIf (cfg.devices.disk != {}) {
-    system.build.formatScript = pkgs.writers.writeBash "disko-create" ''
+    system.build.formatScript = pkgs.writers.writeBashBin "disko-create" ''
```

Comment:
If we switch to this, we should make new attributes and make the old "script" ones execute those with a warning using `lib.warn`. This would give us time to migrate tools like nixos-anywhere.

Reply:
Also `diskoNoDeps` is the only thing used by nixos-anywhere, we still have nixpkgs with the old disko cli around.

Reply:
Maybe something like:

- `diskoFormat`, `diskoMount`, `diskoRecreate`, `diskoRecreateNoDeps`